packages/next/build/analysis/get-page-static-info.ts

```diff
@@ -218,3 +231,19 @@ function warnAboutExperimentalEdgeApiFunctions() {
 }
 
 let warnedAboutExperimentalEdgeApiFunctions = false
+
+const warnedUnsupportedValueMap = new Map<string, boolean>()
+function warnAboutUnsupportedValue(
+  pageFilePath: string,
+  page: string | undefined
+) {
+  if (warnedUnsupportedValueMap.has(pageFilePath)) {
+    return
+  }
+  Log.warn(
+    `You have exported a \`config\` field in ${
+      page ? `route "${page}"` : `"${pageFilePath}"`
+    } that Next.js can't recognize, so it will be ignored`
```

Comment:
Can we mention the specific field that we couldn't analyze? 

Also can we ensure we link to an err.sh document here potentially this [existing doc](https://github.com/vercel/next.js/blob/canary/errors/invalid-page-config.md)

Reply:
> Can we mention the specific field that we couldn't analyze?

The `UnsupportedValueError` is thrown directly when encounter invalid/unknown/unsupported syntax without more details.

Maybe I could implement it in the future PR.

> Also can we ensure we link to an err.sh document here potentially this [existing doc](https://github.com/vercel/next.js/blob/canary/errors/invalid-page-config.md)

Sure! I will update the existing doc and link to it.

Reply:
Fixed in [`d7a713c` (#38750)](https://github.com/vercel/next.js/pull/38750/commits/d7a713c906bd903f20d56bfc7717cf1fab663347)